Allow tests to work in an out-of-source build.

--- libsrc2/test/testminctools.sh	2012-09-21 20:45:06.000000000 +0200
+++ libsrc2/test/testminctools.sh	2015-05-13 12:00:44.537446737 +0200
@@ -4,7 +4,7 @@
 
 set -e
 
-../../mincconvert ../../volume_io/Testing/t3_grid_0.mnc t3_grid_0_2.mnc -2 -clobber && echo "Converted file to MINC2.0 format." || exit 1
+../../mincconvert "$srcdir"/../../volume_io/Testing/t3_grid_0.mnc t3_grid_0_2.mnc -2 -clobber && echo "Converted file to MINC2.0 format." || exit 1
  
 ../../mincinfo -minc_version -image_info t3_grid_0_2.mnc
 
@@ -20,7 +20,7 @@
 ../../mincstats -mean -std t32_added.mnc || exit 1
 
 echo "Resampling file with t1.xfm transform"
-../../mincresample -transform ../../volume_io/Testing/t1.xfm -tfm_input_sampling t32_added.mnc t32_transformed.mnc -clobber || exit 1
+../../mincresample -transform "$srcdir"/../../volume_io/Testing/t1.xfm -tfm_input_sampling t32_added.mnc t32_transformed.mnc -clobber || exit 1
 
 echo "Transformed file"
 ../../mincinfo t32_transformed.mnc
--- testdir/run_test_progs.sh	2012-09-21 20:45:07.000000000 +0200
+++ testdir/run_test_progs.sh	2015-05-13 11:49:27.358011733 +0200
@@ -8,5 +8,5 @@
 PATH=${root}:${progs}/mincdiff::${progs}/mincpik:${progs}/mincheader:${progs}/minchistory:${progs}/mincview:${PATH}
 export PATH
 
-mincheader icv.mnc > /dev/null
-mincdiff icv.mnc icv.mnc > /dev/null
+mincheader "$srcdir"/icv.mnc > /dev/null
+mincdiff "$srcdir"/icv.mnc "$srcdir"/icv.mnc > /dev/null
